Fix typo in ActionMailbox::InboundEmail::MessageId [ci skip]

giraffate opened the pull request:
### Summary
Fix `a Active Storage` -> `an Active Storage`

eugeneius commented on actionmailbox/app/models/action_mailbox/inbound_email/message_id.rb:
```diff
@@ -10,7 +10,7 @@ module ActionMailbox::InboundEmail::MessageId
 
   class_methods do
-    # Create a new +InboundEmail+ from the raw +source+ of the email, which be uploaded as a Active Storage
+    # Create a new +InboundEmail+ from the raw +source+ of the email, which be uploaded as an Active Storage
```
> which be uploaded

Could you fix the grammar here too?

giraffate replied:
Is this grammar ok?
`which be uploaded` -> `which is uploaded`

giraffate committed:
Fix typo in ActionMailbox::InboundEmail::MessageId [ci skip]

- `a Active Storage` -> `an Active Storage`
- `which be uploaded` -> `which is uploaded`

eugeneius commented:
Thanks!
